.github/workflows/check-copyright.yml

```
@@ -0,0 +1,18 @@
+name: Copyright check
+
+on: [pull_request]
```

Do we need to execute it on every PR, or PRs to the master is enough? I would also add the `workflow_dispatch` trigger so that this task could be executed manually if needed.

```suggestion
on:
  pull_request:
    branches: [ master ]
  workflow_dispatch:
```